src/components/CardList/index.jsx

```diff
@@ -1,30 +1,29 @@
-import React, { useEffect, useState } from 'react';
+import React, { useEffect, useState, useRef } from 'react';
 import Movie from '../Card';
-import fetchFrom from '../../services';
+import { fetchMovieByGenre, fetchPopular } from '../../services';
 import './styles.css';
 
 function CardList({
-  id,
   title,
-  url,
   options,
```

Comment:
in my opinion it will make it more readable to have the nested options as individual component props, so i can have a quicker understanding of what is gonna do the component